Batteries/Data/HashMap/Basic.lean

```diff
@@ -482,7 +252,7 @@ def toArray (self : HashMap α β) : Array (α × β) :=
   self.fold (init := #[]) fun r k v => r.push (k, v)
 
 /-- The number of buckets in the hash map. -/
-def numBuckets (self : HashMap α β) : Nat := self.1.buckets.1.size
+def numBuckets (self : HashMap α β) : Nat := self.1.1.buckets.size
```

Comment:
It doesn't really make a difference, but rather than reaching into the guts of the hash map you could invoke `Std.HashMap.Internal.numBuckets` here.